ext/intl/tests/IntlBidi/IntlBidi___construct_basic.phpt

```diff
@@ -0,0 +1,14 @@
+--TEST--
+Regression test for the UBA implementation.
+--CREDITS--
+Timo Scholz <timo.scholz@setasign.com>
+--SKIPIF--
+<?php if ( !extension_loaded('intl') ) print 'skip'; ?>
+<?php if ( !extension_loaded('mbstring')) print 'skip'; ?>
```

Comment:
This is superfluous for this test.

Reply:
Its not superflous, since its testing the initialization and collection of the object.

With this test we cover bidi_object_from_zend_object and bidi_object_to_zend_object.
(We make sure that the struct and the functions work together without crashing)
This test is primarily to make sure that this structure is not faulty.

Reply:
And it requires mbstring?

Reply:
Actually it does not require mbstring, i changed that.